roles/download/defaults/main.yml

```diff
@@ -120,114 +93,97 @@ kubeadm_binary_checksum: "{{ kubeadm_checksums[kubeadm_version] }}"
-metrics_server_version: "v0.3.1"
+ingress_nginx_controller_image_repo: "quay.io/kubernetes-ingress-controller/nginx-ingress-controller"
+ingress_nginx_controller_image_tag: "{{ ingress_nginx_controller_version }}"
+install_socat_image_repo: "xueshanf/install-socat"
+install_socat_image_tag: "{{ socat_image_version }}"
+kube_router_image_repo: "cloudnativelabs/kube-router"
+kube_router_image_tag: "{{ kube_router_version }}"
+kubedns_image_repo: "gcr.io/google_containers/k8s-dns-kube-dns-{{ image_arch }}"
+kubedns_image_tag: "{{ kubedns_version }}"
+local_volume_provisioner_image_repo: "quay.io/external_storage/local-volume-provisioner"
+local_volume_provisioner_image_tag: "{{ local_volume_provisioner_version }}"
 metrics_server_image_repo: "k8s.gcr.io/metrics-server-amd64"
 metrics_server_image_tag: "{{ metrics_server_version }}"
-local_volume_provisioner_image_repo: "quay.io/external_storage/local-volume-provisioner"
-local_volume_provisioner_image_tag: "v2.1.0"
-cephfs_provisioner_image_repo: "quay.io/external_storage/cephfs-provisioner"
-cephfs_provisioner_image_tag: "v2.1.0-k8s1.11"
-ingress_nginx_controller_image_repo: "quay.io/kubernetes-ingress-controller/nginx-ingress-controller"
-ingress_nginx_controller_image_tag: "0.21.0"
-cert_manager_version: "v0.5.2"
-cert_manager_controller_image_repo: "quay.io/jetstack/cert-manager-controller"
-cert_manager_controller_image_tag: "{{ cert_manager_version }}"
-addon_resizer_version: "1.8.3"
-addon_resizer_image_repo: "k8s.gcr.io/addon-resizer"
-addon_resizer_image_tag: "{{ addon_resizer_version }}"
-
-dashboard_image_repo: "gcr.io/google_containers/kubernetes-dashboard-{{ image_arch }}"
-dashboard_image_tag: "v1.10.0"
+multus_image_repo: "docker.io/nfvpe/multus"
+multus_image_tag: "{{ multus_version }}"
+netcheck_agent_image_repo: "quay.io/l23network/k8s-netchecker-agent"
+netcheck_agent_image_tag: "{{ netcheck_version }}"
+netcheck_server_image_repo: "quay.io/l23network/k8s-netchecker-server"
+netcheck_server_image_tag: "{{ netcheck_version }}"
+nginx_image_repo: "nginx"
+nginx_image_tag: "{{ nginx_version }}"
+pod_infra_image_repo: "gcr.io/google_containers/pause-{{ image_arch }}"
+pod_infra_image_tag: "{{ pod_infra_version }}"
+registry_image_repo: "registry"
+registry_image_tag: "{{ registry_version }}"
+registry_proxy_image_repo: "gcr.io/google_containers/kube-registry-proxy"
+registry_proxy_image_tag: "{{ registry_proxy_version }}"
+tiller_image_repo: "gcr.io/kubernetes-helm/tiller"
+tiller_image_tag: "{{ helm_version }}"
+weave_kube_image_repo: "docker.io/weaveworks/weave-kube"
+weave_kube_image_tag: "{{ weave_version }}"
+weave_npc_image_repo: "docker.io/weaveworks/weave-npc"
+weave_npc_image_tag: "{{ weave_version }}"
 
 downloads:
   netcheck_server:
     enabled: "{{ deploy_netchecker }}"
     container: true
     repo: "{{ netcheck_server_image_repo }}"
     tag: "{{ netcheck_server_image_tag }}"
-    sha256: "{{ netcheck_server_digest_checksum|default(None) }}"
```

Comment:
Sha verification should stay

Reply:
Why sha verify when one of the `*_digest_checksum` is not set at all. It does not even work now?

Reply:
that probably a practice to force to set the checksum for every binaries in kubespray

Reply:
instead of 'none' if does not work could be `default(omit)`, and that works for sure